File: app/src/main/java/com/se/dr/bo/seconddrawingboard/wight/MotionAction.java
package com.se.dr.bo.seconddrawingboard.wight;

import android.graphics.Canvas;

public interface MotionAction {
    void actionDown(Canvas canvas, float x, float y);

    void actionMove(Canvas canvas, float x, float y);

    void actionUp(Canvas canvas, float x, float y);
}

File: app/src/main/java/com/se/dr/bo/seconddrawingboard/wight/MotionAction.kt
package com.se.dr.bo.seconddrawingboard.wight

import android.graphics.Canvas

interface MotionAction {
    fun actionDown(canvas: Canvas?, x: Float, y: Float)
    fun actionMove(canvas: Canvas?, x: Float, y: Float)
    fun actionUp(canvas: Canvas?, x: Float, y: Float)
}
